src/ai/backend/test/testcases/auth/testspecs.py

```diff
@@ -0,0 +1,12 @@
+from ai.backend.test.templates.auth.login import LoginTemplate
+from ai.backend.test.templates.template import BasicTestTemplate, NopTestCode
+from ai.backend.test.testcases.testcases import TestSpec, TestTag
+
+AUTH_TEST_SPECS = {
+    "login": TestSpec(
+        name="login",
+        description="Test user login functionality.",
+        tags={TestTag.WEBSERVER, TestTag.AUTH},
+        template=LoginTemplate(template=BasicTestTemplate(NopTestCode())),
```

Comment:
It's more annoying than I thought to attach the basic template every time...
I will think about how to improve this part later.